scripts/tests_analyzer/compare_coderabbit_decisions.py

```
@@ -0,0 +1,689 @@
+
+from simple_logger.logger import get_logger
+
+# Configure logging
+logger = get_logger(name=__name__, level=logging.INFO)
+
+
+def get_default_repo() -> str:
+    """Try to detect repo from git remote, fallback to hardcoded default."""
+    try:
+        result = subprocess.run(
+            ["git", "config", "--get", "remote.origin.url"],
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)